### PR TITLE
Added mouseover text to configuration module project and subproject labels

### PR DESCRIPTION
**Pull request by colezweber**
This PR addresses https://redmine.cbrain.mcgill.ca/issues/11343
Added mouseover text (tooltip) to the Project and Subproject sections of the Configuration Module.

### CODE REVIEW

**Comment by kongtiaowang** [comment]
@colezweber Why you put the "Help! I don't understand Travis!" here?

**Comment by colezweber** [comment]
After the "Adjusted EDC description" commit, it didn't pass with Travis and I couldn't find out why because it didn't flag any errors with my code, so I re-ran it and it eventually passed

**Comment by zaliqarosli** [comment]
new issue: mouseover tooltip does not show for 'New ProjectID' tab

**Comment by zaliqarosli** [comment]
needs retesting

File: modules/configuration/templates/form_subproject.tpl [hunk 1]
```diff
@@ -23,31 +23,39 @@
             <fieldset>
                 <input type="hidden" name="subprojectID" value="{$subprojectID}" class="subprojectID">
                 <div class="form-group">
-                    <label class="col-sm-12 col-md-4">Subproject Title</label>
-                    <div class="col-sm-12 col-md-8">
+                    <div class="col-sm-12 col-md-3" data-toggle="tooltip" data-placement="right" title="{'Full descriptive title of the subproject'}">
+                        <label class="col-sm-12 control-label">Subproject Name</label>
+                    </div>
+                    <div class="col-sm-12 col-md-7">
```
Comment (zaliqarosli) [inline modules/configuration/templates/form_subproject.tpl]: change these 7s to 9s

Comment (zaliqarosli) [reply modules/configuration/templates/form_subproject.tpl]: and this (https://github.com/colezweber/Loris/blob/0117bd2be8b5e4e5b86e66124245bdd43a81b427/modules/configuration/templates/form_subproject.tpl#L16) to 7

**Comment by zaliqarosli** [comment]
code looks good, needs testing for functionality! @miebeers

**Pull request merged**